docs/pages/1.introduction.stories.mdx

```diff
@@ -47,6 +47,10 @@ Here's a pen, and some other blog post text
 
 <Box sx={{ height: '24px' }} />
 
+<PayWithStripe />
```

Comment:
Is this a rebase issue. Did you mean to commit this file?

Reply:
This was a rebase, I didn't touch that yo!